localtileserver/tileserver/__init__.py

```diff
@@ -19,3 +21,18 @@
     make_vsi,
     purge_cache,
 )
+
+
+def create_app(url_prefix: str = "/", cors_all: bool = True):
+    try:
+        from localtileserver.tileserver import sentry
+    except ImportError:
+        pass
+    app = Flask(__name__)
+    if cors_all:
+        cors = CORS(app, resources={r"/api/*": {"origins": "*"}})
+    cache.init_app(app)
+    app.register_blueprint(tileserver, url_prefix=url_prefix)
+    app.config.JSONIFY_PRETTYPRINT_REGULAR = True
+    app.config.SWAGGER_UI_DOC_EXPANSION = "list"
+    return app
```

Comment:
This looks like an older duplocicate of:

https://github.com/banesullivan/localtileserver/blob/d5bfa5693b64d857051b47cc5999b8de9e0ef58e/localtileserver/tileserver/application.py#L15

If having `cors_all` default to True is preferred, we can make this an environment variable that you all can set in your docker image